procrastinate/utils.py

```diff
@@ -432,3 +434,17 @@ def async_context_decorator(func: Callable) -> Callable:
         return contextlib2.asynccontextmanager(func)
     else:
         return contextlib.asynccontextmanager(func)
+
+
+class TimeDeltaParams(TypedDict):
+    weeks: NotRequired[int]
+    days: NotRequired[int]
+    hours: NotRequired[int]
+    minutes: NotRequired[int]
+    seconds: NotRequired[int]
+    milliseconds: NotRequired[int]
+    microseconds: NotRequired[int]
```

Comment:
(either this or `types.py` as we're using it in multiple places)

Reply:
Yes, types.py sounds like a better location.